tool/core.py

```diff
@@ -17,9 +17,11 @@ def get_service(self, service_name):
         else:
             ServiceClass = map[service_name]
         configuration = self.conf.c[service_name]
+        configuration["full_repo_name"] = repo
         return ServiceClass(**configuration)
 
-    def guess_service(self, remote):
+    # TODO: change to remote=None and iterate over all remotes
+    def guess_service(self, remote="origin"):
```

Comment:
For now, I would use `remote="upstream"` to see the same result for both cases:
`tool list-prs` and `tool list-prs user-cont/tool`.